backend/requirements.txt

```diff
@@ -5,4 +5,5 @@ djangorestframework==3.12.4
 djangorestframework-jsonapi
 djangorestframework-simplejwt==5.0.0
 django_rest_passwordreset
-django-cors-headers
\ No newline at end of file
+django-cors-headers
+django-activity-stream
```

Comment:
We do not need this either, same as above

Reply:
Removed.